.changeset/hip-gorillas-joke.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Need additional changeset for segment tracking calls.

Reply:
I know but wasn't sure if we wanted this to be major as our first official release, or if we were going to continue with minor updates. What do you think?

Reply:
I think we should hold off. I'm going to have a few more things to change from integrating this into cloud.

I think once we go live on cloud-dev we can bump this to 1.0.0.

Reply:
👍